src/pages-and-resources/discussions/app-list/AppCard.jsx

```diff
@@ -49,6 +54,22 @@ function AppCard({
         </div>
         <Card.Subtitle className="mb-3 text-muted">{supportText}</Card.Subtitle>
         <Card.Text>{intl.formatMessage(messages[`appDescription-${app.id}`])}</Card.Text>
+        <ExtraSmall>
+          <Collapsible title={intl.formatMessage(messages.supportedFeatureList)} styling="basic">
+            {features && features.map((feature) => (
+              <div key={`collapsible-${app.id}&${feature.id}`}>
```

Comment:
I think we should move this to its own component and use it here like. Name the component that makes more sense to you. 

```
<ExtraSmall>
  <FeaturesCollapsible /> 
</ExtraSmall>
```